src/chttpd_db.erl

```diff
@@ -74,6 +86,8 @@ handle_changes_req1(#httpd{}=Req, Db) ->
     ChangesArgs = Args0#changes_args{
         filter_fun = couch_changes:configure_filter(Raw, Style, Req, Db)
     },
+    % Default to ~filling the payload of a standard Ethernet frame
+    Max = config:get_integer("httpd", "chunked_response_buffer", 1490),
```

Comment:
We should figure out what the optimal size really is; filling an Ethernet frame was almost an arbitrary choice on my part. We did see a larger increase in throughput with a 64KB buffer. If the frame size really does turn out to be a good target then yes, we should either programmatically discover the MTU or document the way to optimize this.

I chose 1490 instead of 1500 because mochiweb needs a couple of bytes to declare the size of the chunk and to add a newline or two. I didn't compute exactly how many bytes are required for that "envelope"; it's probably a bit less than 10 but I figured it was better to err on the conservative side than to create a new, nearly-empty packet.